src/domain/bitcoin/bria/index.types.d.ts

```diff
@@ -0,0 +1,39 @@
+type ClientReadableStream<T> = import("@grpc/grpc-js").ClientReadableStream<T>
+
+type UtxoDetectedEvent = {
```

Comment:
I would've leaned more to having these as service types instead of being here in the domain, but I needed `UtxoEvent` to pass in as a type argument to `ClientReadableStream` in the service type definition lower down